core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -6186,6 +6186,31 @@ public void subTestIntervalDayFailsValidation() {
     assertEquals(linux(sqlNodeVisited1.toString()), str1);
   }
 
+  @Test void testSqlDeleteSqlBasicCallToString() throws Exception {
```

Comment:
the name of the test is misleading. you are not calling toString on a SqlBasicCall.